proto/device_sync/content.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package xmtp.device_sync.content;
+
+import "mls/message_contents/content.proto";
+import "device_sync/consent_backup.proto";
+
+message DeviceSyncContent {
+  oneof content {
+    xmtp.mls.message_contents.DeviceSyncRequest request = 1;
+    xmtp.mls.message_contents.DeviceSyncReply payload = 2;
+    AcknowledgeKind acknowledge = 3;
+    PreferenceUpdates preference_updates = 4;
+  }
+}
+
+message AcknowledgeKind {
+  oneof kind {
+    SyncGroupPresence sync_group_presence = 1;
+    RequestAcknowledge request = 2;
+  }
+}
+
+message SyncGroupPresence {
+}
+
+message RequestAcknowledge {
+  string request_id = 1;
+}
```

Comment:
Would it make sense to collapse this all into a single `DeviceSyncAcknowledge` message with an optional `request_id` (or a `request_id` of 0 if it was an implicit request)?

If we're using conversation stitching (abstracting multiple sync groups so that we can pretend there is a single long-lived sync group), I'm also wondering if we could do away with implicit requests and have all requests be explicit?